### PR TITLE
Handle missing time-save file and other fixes

### PR DESCRIPTION
**Pull request by alanggreen**
1. When time is resumed, countdown should in green not yellow
1. Handle use-case where time-save file is missing
1. Clear time-save on app exit and on stop otherwise time will restart
1. In MATCH_BUFFER_TIME start countdown from 1 otherwise get negative time on display

### CODE REVIEW

File: server/time-saver.js [hunk 1]
```diff
@@ -15,16 +15,26 @@ exports.TimeSaver = class {
...
-      }
-      return null
-    })
+    return readFile(FILE_PATH, 'utf-8')
+      .then(time => {
+        time = JSON.parse(time)
+        if (time) {
+          return new Date(parseInt(time))
+        }
+        return null
```
Comment (2roy999) [inline server/time-saver.js]: redundant - you can remove it.

File: server/time-saver.js [hunk 2]
```diff
@@ -15,16 +15,26 @@ exports.TimeSaver = class {
...
+      })
+      .catch(err => {
+        if (err.code != "ENOENT") {
+          logger.warn(`Error reading TimeSaver file ${FILE_PATH}`)
+          throw err
+        }
+        else {
+          return null
```
Comment (2roy999) [inline server/time-saver.js]: redundant you can remove it

File: server/index.js [hunk 3]
```diff
@@ -40,5 +40,16 @@ app.listen(process.env.PORT, () => logger.info(`Listening on port ${process.env.
 
 process.on('SIGINT', () => {
   logger.info('Process received SIGINT: shutting down')
+  timeSaver.clearTime();
+  process.exit(1)
```
Comment (2roy999) [inline server/index.js]: change to exit(130). The practice when exiting cause of system signal is to exit with error code `128 + n` where `n` is the signal number.

File: server/time-saver.js (hunk 1 above)
Comment (alanggreen) [reply server/time-saver.js]: Done

File: server/index.js (hunk 3 above)
Comment (alanggreen) [reply server/index.js]: Done

File: server/time-saver.js (hunk 2 above)
Comment (alanggreen) [reply server/time-saver.js]: Done

**Pull request merged**